massa-execution-worker/src/interface_impl.rs

```diff
@@ -534,6 +534,10 @@ impl Interface for InterfaceImpl {
         Ok(addr.to_string())
     }
 
+    fn validate_address(&self, address: &str) -> Result<bool> {
+        Ok(massa_models::address::Address::from_str(address).is_err())
```

Comment:
Wondering if it should not be is_ok() instead of is_err(). validate should return true if all went well no?

Reply:
Oops, I changed it but forgot to push it